nomad/poc-2.hcl

```diff
@@ -24,125 +40,104 @@ job "poc-2" {
-                    --override-input versions/holochain 'github:holochain/holochain/a585a619d68bd47d2e995a773cdf76dea08fdca3' \
-                    --command \
-                    hc-run-local-services \
-                        --bootstrap-interface 0.0.0.0 \
-                        --bootstrap-port ''${NOMAD_PORT_holochainBootstrap} \
-                        --signal-interfaces 0.0.0.0 \
-                        --signal-port ''${NOMAD_PORT_holochainSignal}
-                ENDOFSCRIPT
+                args = ["bash", "-c",
+                    <<ENDOFSCRIPT
+                    set -xeu
+
+                    source local/init.sh
+                    env
+
+                    export TEST_SHARED_VALUES_REMOTEV1_ROLE="server"
+                    export TEST_SHARED_VALUES_REMOTEV1_URL="ws://${NOMAD_HOST_IP_holochainTestRemoteV1}:${NOMAD_HOST_PORT_holochainTestRemoteV1}"
+
+                    nix develop -vL .#coreDev --command \
+                        cargo nextest run --locked -p holochain_test_utils --no-capture --features slow_tests --status-level=pass --retries=99999999 discovery_distributed
+                    ENDOFSCRIPT
                 ]
             }
         }
+
     }
 
+    group "clients" {
+        count = var.agents
 
+        restart {
+            attempts = 0
+            mode = "fail"
+        }
 
-    task "holochain-conductor" {
         constraint {
-            attribute = "${meta.features}"
-            operator = "set_contains"
-            value = "ipv4-nat,nix"
+            distinct_hosts = true
         }
 
-        template {
-            data = <<EOH
-                {{ range nomadService "holochain-boostrap" }}
-                BOOTSTRAP_HOST="{{ .Address }}"
-                BOOTSTRAP_PORT="{{ .Port }}"
-                {{ end }}
-                {{ range nomadService "holochain-signal" }}
-                SIGNAL_HOST="{{ .Address }}"
-                SIGNAL_PORT="{{ .Port }}"
-                {{ end }}
-                EOH
-            destination = "local/env.txt"
-            env         = true
-        }
+        task "test-remotev1-distributed-clients" {
+            constraint {
+                attribute = "${meta.features}"
+                operator = "set_contains"
+                value = "ipv4-nat,nix"
+            }
+
+            env {
+                GIT_URL = "${var.GIT_URL}"
+                GIT_BRANCH = "${var.GIT_BRANCH}"
+            }
+
+            template {
+                data = <<-EOH
+                    {{ range nomadService "holochainTestRemoteV1" }}
+                    TEST_SHARED_VALUES_REMOTEV1_ROLE="client"
+                    TEST_SHARED_VALUES_REMOTEV1_URL="ws://{{ .Address }}:{{ .Port }}"
+                    TEST_AGENT_READINESS_REQUIRED_AGENTS=${var.agents}
+                    TEST_AGENT_READINESS_TIMEOUT_SECS=${var.agents_readiness_timeout_secs}
+                    {{ end }}
+                    EOH
+                destination = "local/env.txt"
```

Comment:
Does the test know to look for this? I don't see what's reading it otherwise

Reply:
this is handled by nomad automatically